### PR TITLE
Like Unlike Activity

### PR DESCRIPTION
**Pull request by SSalekin**

### CODE REVIEW

**Comment by Rownaknira** [comment]
ok

**Comment by abdullahallmehedi** [comment]
ok

File: app/views/layouts/_user_header.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <li><%= link_to t(".books"), books_path %></li>
-<li><%= link_to t(".categories"), categories_path %></li>
+<li><%= link_to t(".categories"), "#" %></li>
```
Comment (noor-framgia) [inline app/views/layouts/_user_header.html.erb]: resloved, so add categories_path

**Comment by Rownaknira** [comment]
ok

**Comment by noor-framgia** [comment]
ok

**Comment by abdullahallmehedi** [comment]
ok

**Pull request merged**